app/src/androidTest/java/com/github/multimatum_team/multimatum/util/MockAuthRepository.kt

```diff
@@ -11,36 +11,35 @@ import com.github.multimatum_team.multimatum.repository.DeadlineRepository
 /**
  * Defines a dummy AuthRepository that works locally by generating unique anonymous users.
  */
-class MockAuthRepository : AuthRepository {
+class MockAuthRepository : AuthRepository() {
     private var uniqueIDSupply = 0
 
-    private var user: User = generateNewAnonymousUser()
+    init {
+        _user = generateNewAnonymousUser()
+    }
 
     private val updateListeners: MutableList<(User) -> Unit> =
         mutableListOf()
 
     private fun notifyUpdateListeners() =
-        updateListeners.forEach { it(user) }
+        updateListeners.forEach { it(_user) }
```

Comment:
Why the change of name ?

Reply:
This is because the AuthRepository was refactored. Instead of creating an anonymous user in the `getCurrentUser()` method, there is now only a `getUser()` getter which returns the underlying `_user` that is set in the constructor.